### PR TITLE
Updated ta-IN

### PR DESCRIPTION
**Pull request by kkavya30**
Made corrections and updated list. New contributor!!!

### CODE REVIEW

**Comment by guyluz11** [comment]
Hi
It looks like you have used `;` at the end of line, please convert it into `,`

File: assets/translations/ta-IN.json [hunk 1]
```diff
@@ -102,11 +102,11 @@
   "Do_not_need": "தேவையில்லை",
   "Finish": "முடிக்கவும்",
   "Select_Purposes_Of_The_Area": "பகுதியின் நோக்கங்களைத் தேர்ந்தெடுக்கவும்"
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:104]: Please add `,` at the end of the line.

File: assets/translations/ta-IN.json [hunk 2]
```diff
@@ -102,11 +102,11 @@
...
   "Stopping_all_blinds": "திரைசீலைகளை நிருத்துவது",
-  "Pulling_down_all_blinds": "திரைசீலைகளை இழுப்பது";
-  "Turning_On_boiler": "கொதிக்கலனை ஆரம்பிப்பது";
-  "Turning_Off_boiler": "கொதிக்கலனை அணைப்பது";
-  "Waiting_for_user_to_get_connected_to_WiFi": "பயனாளி வைஃபை இணைய காத்திருக்கிறது";
+  "Pulling_down_all_blinds": "திரைசீலைகளை இழுப்பது",
+  "Turning_On_boiler": "கொதிக்கலனை ஆரம்பிப்பது",
+  "Turning_Off_boiler": "கொதிக்கலனை அணைப்பது",
+  "Waiting_for_user_to_get_connected_to_WiFi": "பயனாளி வைஃபை இணைய காத்திருக்கிறது",
   "Can't_add_unsupported_device": "ஆதரிக்கப்படாத சாதனத்தைச் சேர்க்க முடியாது"
```
Comment (guyluz11) [inline assets/translations/ta-IN.json:111]: Please don't add `,` at the end of the line. (keep it like it is now)

Comment (kkavya30) [reply assets/translations/ta-IN.json:111]: Hello, I don't get what you mean by that. Do you want it removed or do you want it removed or change it to something else?

Comment (guyluz11) [reply assets/translations/ta-IN.json:111]: No change is good.

**Comment by guyluz11** [comment]
Thanks for the pr @kkavya30 
And a happy Hacktoberfest :D

**Pull request merged**